strings.txt

```diff
@@ -16139,3 +16139,57 @@
     fr = %1$d de %2$d
     ja = %2$d中%1$d
     vi = %1$d trên %2$d
+
+  [download_over_mobile_header]
+    en = Download by using a cellular network connection?
```

Comment:
Есть примеры в стандартных приложениях от apple и google? я не видел. Вопрос должен быть вопросом, а не утверждением.

> On Apr 1, 2016, at 14:27, therearesomewhocallmetim <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/2675#discussion_r58193124>:
> 
> > @@ -16139,3 +16139,57 @@
> >      fr = %1$d de %2$d
> >      ja = %2$d中%1$d
> >      vi = %1$d trên %2$d
> > +
> > +  [download_over_mobile_header]
> > +    en = Download by using a cellular network connection?
> Алекс, вполне допустимо использовать в качестве вопросительного.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2675/files/0e007d81ca5291f9ae45538d8ce3f73e38dd4e1b#r58193124>